snappy_wrappers/wrappers/mantis/run/wrapper.py

```diff
@@ -33,7 +33,7 @@
 # but should also work for reasonable deep WGS according to them.
 # https://github.com/OSU-SRLab/MANTIS/issues/25
 
-python /fast/groups/cubi/projects/biotools/Mantis/MANTIS/mantis.py \
+python /fast/groups/cubi/work/projects/biotools/Mantis/MANTIS/mantis.py \
```

Comment:
`mantis` should be in the `PATH`, as it is installed in the conda environment.

Also, please don't mix different work packages. I know you have used this pull request to fix minor issues with the TMB step, but I'd like to minimise these event.